diff --git a/node_modules/@web3-onboard/wagmi/dist/index.js b/node_modules/@web3-onboard/wagmi/dist/index.js
index 009072a..d32779a 100644
--- a/node_modules/@web3-onboard/wagmi/dist/index.js
+++ b/node_modules/@web3-onboard/wagmi/dist/index.js
@@ -1,10 +1,9 @@
-import { createConfig, createConnector, getConnectors } from '@wagmi/core';
-import { createWagmiChains, createWalletId } from './utils';
-import { connect as wagmiConnect } from '@wagmi/core';
-import { validateWagmiInit } from './validation';
-import EventEmitter from 'eventemitter3';
-import { toHex, fromHex, UserRejectedRequestError, ResourceUnavailableRpcError } from 'viem';
+import { createConfig, createConnector, getConnectors, connect as wagmiConnect } from '@wagmi/core';
 import { ProviderRpcErrorCode } from '@web3-onboard/common';
+import EventEmitter from 'eventemitter3';
+import { fromHex, ResourceUnavailableRpcError, toHex, UserRejectedRequestError } from 'viem';
+import { createWagmiChains, createWalletId } from './utils.js';
+import { validateWagmiInit } from './validation.js';
 export * from '@wagmi/core';
 const wagmiConnectorFn = {};
 const transports = {};
